pkg/util/os.go

```diff
@@ -0,0 +1,17 @@
+package util
+
+import (
+	"os"
+)
+
+// Create a directory including its parents.
```

Comment:
I was confused for a moment since Mkdir is called both in DumpCA() and RunServer()

Might be worth mentioning the behaviour in the case where the directory already exists in the comment.
```suggestion
// Create a directory including its parents. Returns nil if directory already exists.
```

Reply:
It is "mkdir -p", so it does return 0/nil :) Will add the comment, thank you.